src/installing-workflow/index.md

```diff
@@ -83,6 +83,8 @@ $ helmc install workflow-v2.8.0               # injects resources into
 
     	$ helm install deis/workflow --version=v2.8.0 --namespace=deis -f <optional values file>  # injects resources into your cluster
 
+  See also our section on [Workflow chart provenance](workflow-helm-charts.md#chart-provenance)
+
```

Comment:
@kmala good point -- actually, hoping a follow-up PR to address https://github.com/deis/workflow/issues/570 will remove/condense all the other (duplicated) sections, so perhaps moot by then.